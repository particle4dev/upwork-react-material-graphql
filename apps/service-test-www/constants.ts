import keys from 'lodash/keys';

export const ROUTER = {
  SEARCH: {
    label: 'Search',
    path: '/search'
  },
  BROWSE: {
    label: 'Browse',
    path: '/browse'
  },
  US: {
    label: 'About Us',
    path: '/us'
  },
  HOME: {
    label: 'Home',
    path: '/'
  }
};

export const ALL_ROUTER: string[] = keys(ROUTER);

export const PORT = parseInt(process.env.PORT, 10) || 3000;
