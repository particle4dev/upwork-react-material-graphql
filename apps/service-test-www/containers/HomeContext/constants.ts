const createSymbol = (name: string) => `components/HomeContext/${name}`;

export const UPDATE_DETAIL_DIALOG = createSymbol('UPDATE_DETAIL_DIALOG');

export const ADD_NEW_ROW = createSymbol('ADD_NEW_ROW');

export const REMOVE_ROW = createSymbol('REMOVE_NEW_ROW');

export const SET_SELECT_ALL = createSymbol('SET_SELECT_ALL');

export const UPDATE_DATA_ID = createSymbol('UPDATE_DATA_ID');

export const UPDATE_PREVIEW_DIALOG = createSymbol('UPDATE_PREVIEW_DIALOG');
