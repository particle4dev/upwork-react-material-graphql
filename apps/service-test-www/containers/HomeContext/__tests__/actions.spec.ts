import {
  updateDetailDialog
} from '../actions';
import { UPDATE_DETAIL_DIALOG } from '../constants';

describe('containers/HomeContext/actions/updateDetailDialog', () => {
  const status = true;
  it('should updateDetailDialog should create updateDetailDialog action', () => {
    expect(updateDetailDialog(status)).toMatchSnapshot();
  });

  it('should return the correct type and the passed name', () => {
    const expectedResult = {
      type: UPDATE_DETAIL_DIALOG,
      payload: {
        status
      }
    };

    expect(updateDetailDialog(status)).toEqual(expectedResult);
  });
});
