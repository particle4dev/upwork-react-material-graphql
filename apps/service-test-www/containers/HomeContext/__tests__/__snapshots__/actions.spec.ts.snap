// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/HomeContext/actions/updateDetailDialog should updateDetailDialog should create updateDetailDialog action 1`] = `
Object {
  "payload": Object {
    "status": true,
  },
  "type": "components/HomeContext/UPDATE_DETAIL_DIALOG",
}
`;
